src/types.py

```diff
@@ -0,0 +1,62 @@
+from numpy import dtype, array, recarray, uint8, ndarray
+from dataclasses import dataclass
+from numpy.typing import ArrayLike
+
+#FIXME: import flags from argparse for colormode
+
+# RGBA Color
+@dataclass
+class RGBA():
+    r:uint8
+    g:uint8
+    b:uint8
+    a:uint8
+    
+    def get_all(self)-> ndarray:
+        return array((self.r,self.g,self.b,self.a))
+
+@dataclass
+class Vertex():
+    x:ndarray
+    y:ndarray
+    def pairs(self,):
+        """
+        Get x,y pairs
+        """
+        return zip(self.x,self.y)
+    
+
+@dataclass
+class Solution():
```

Comment:
`sequence: list[Solution]` seems to imply that this an abstraction for a single polygon? Or did you mean a solution with multiple polygons in sequence?

Reply:
I guess I was thinking polygon when I originally wrote the class -- How do you think we should move forward with this?

Reply:
I definitely don't have an answer but I think if you take a look at the code in `polygon-vis` branch, you might get a sense of what might be useful.